gift-mode.el

```diff
@@ -42,6 +42,50 @@
 (require 'outline)
 (require 'newcomment)
 
+
+(defvar gift-imenu-comments-regexp "\\(\\(^\\s-*//.*$\\)\\|\\(^\\s-*$\\)\\)+")
+
+(defvar gift-imenu-question-regexp "^\\(\\([^{}]\\)+\\){" )
+
+(defvar gift-imenu-title-regexp "::\\(.*\\)::" )
+
+
+(defun gift-imenu-log (msg &rest args)
```

Comment:
was this just for debugging?

Reply:
yes, it was.
This function is removed in the last commit.